### PR TITLE
Adds Sickly Pears

### PR DESCRIPTION
**Pull request by mithsaxel**
[ASS-JAM] 
## About the PR 
It adds a pear that infects you with a disease that eventually causes everything you try to pick up to melt. 

## Why's this needed? 
ASS JAM fun for botany.

### CODE REVIEW

**Comment by ZeWaka** [comment]
This will straight up delete many things, including nukes.

**Comment by mithsaxel** [comment]
As I'm testing it doesn't appear to be melting nukes, because they are machinery, and not items. And in my testing I have been able to mutate the pears to get it.

**Comment by ZeWaka** [comment]
> As I'm testing it doesn't appear to be melting nukes, because they are machinery, and not items. And in my testing I have been able to mutate the pears to get it.

Okay, nukes can't be melted, read the code wrong. However, any item can. Things like the nuclear disk, people's IDs, and many other important items can just be deleted instantly.

**Comment by zjdtmkhzt** [comment]
To be honest, I found this PR really funny, and I feel like it might create some very funny situations on ass day without being too disruptive, with some changes.

I don't really see the melting items part as too much of an issue, as long as it only applies to items you can touch and pick up. If you were able to take a crewmembers ID, you could already run off with it. I feel like destroying it after you take it isn't really much worse. As for the nuke disk, I was under the impression that it respawns when it is destroyed anyway?

From what I can see, the hot arms are unable to be removed, which I would advise against, I feel like it would be funnier if you could cure the condition by replacing your arms, with someone else possibly taking them instead.
In addition, maybe make burning people on interacting with them only a relatively small chance (10-20%?). I feel like that would make it so people would be more likely to try to do things and fail sometimes, which would be quite funny in a similar way to the clumsy bioeffect.
Lastly, the description of molten items should be changed to respect the hot person's gender.

File: code/modules/medical/diseases/high_fever.dm [hunk 1]
```diff
@@ -22,7 +23,7 @@
 				affected_mob.bodytemperature += 10
 		if(4)
 			if(prob(10))
-				boutput(affected_mob, "<span class='alert'>You're REALLY starting to heat up...</span>")
+				boutput(affected_mob, "<span class='alert'>You're REALLY starhttps://github.com/mithsaxel/goonstation/blob/pear/code/modules/medical/diseases/high_fever.dmting to heat up...</span>")
```
Comment (zjdtmkhzt) [inline code/modules/medical/diseases/high_fever.dm:26]: I assume this was an accidental copying?^^

**Comment by zjdtmkhzt** [comment]
Alright, just tested everything locally, it seems to work for me!
The ability to smash windows and grills should perhaps be removed, since it doesn't thematically fit with extreme heat and is very strong.
The damage on harm intent is very strong right now, it could use a little nerf, I think. (Although the inability to pick up items is a large downside.) I was able to easily fight two monkeys at once with little damage to myself by burning them with my clutch.
Lastly, there are some "items" that are not usually thought of as items, but that can nonetheless be molten in your clutch. Perhaps some of these should have exceptions made to not melt them. These include (probably among other things):
- supply closets & cabinets (probably should be immune)
- intercoms (probably should be immune)
- jukebox (probably fine)
- lollipop bowls (probably fine)
- lava lamps (probably fine)
- gaming kits (probably fine)
- paper bins (probably fine)
I am basing this mainly on the fact that melting cabinets and intercoms are a bit more disruptive, while melting something like a jukebox is pretty funny.

Apart from that, great patch, great changes, keep it up. :)

**Comment by Tarmunora** [comment]
Anything that is anchored could be immune, perhaps

**Comment by zjdtmkhzt** [comment]
Yeah, that's probably a better idea!

**Pull request merged**